tests/conftest.py

```diff
@@ -309,6 +326,18 @@ def tzinfo(request):
     return BasicTimezone(offset=datetime.timedelta(hours=hours_offset), name=request.param)
 
 
+@pytest.fixture(scope="session")
+def tzinfos():
```

Comment:
what's the benefit of having this as a fixture? IISC it's used only in `tz_bots` and could also just be hardcoded there?

Reply:
mm yeah, a bit of a premature optimization at this stage in the PR, but yes I'll keep it mind